SalesforceCommerceCloud/pwa-kit
# [V3] Remove `cross-fetch` from `template-retail-react-app`

# Description

We recently removed the `commerce-api` client code from the template, leaving behind the einstein implementation. Because einstein is only ever used on the client we don't need to use `cross-fetch`, so we are removing it in this PR.

# Types of Changes




# Changes

- Remove `cross-fetch` from useEinstein hook
- Fix tests to mock global fetch.
- Remove dep in package.json and re-lock

# How to Test-Drive This PR

- Run tests, they should pass.
- Preview the template and navigate from page to page, you should see fetch logs for `viewCategory` `viewProduct` etc. This means things are still working.

## General
